assets/components/legal/legalSection/legalSectionContainer.js

```diff
@@ -4,21 +4,14 @@
 
 import { connect } from 'react-redux';
 
-import type { CommonState } from 'helpers/page/commonReducer';
-
 import LegalSection from './legalSection';
 
-
 // ----- State Maps ----- //
 
-function mapStateToProps(state: { common: CommonState }) {
-
-  return {
-    countryGroupId: state.common.internationalisation.countryGroupId,
-  };
-
-}
-
```

Comment:
I'm not sure why the varying format for the mapStateToProps function in different places.  I changed it to this other format bu tI'm not sure whether it loses type safety doing it this way?